modules/local/filter_stats.nf

```diff
@@ -22,15 +22,13 @@ process FILTER_STATS {
     """
     readnumber=\$(wc -l ${reads} | awk '{ print \$1/4 }')
     cat ./filtered.${meta.id}_*.stats | \\
-    tr '\n' ', ' | \\
+    tr '\\n' ', ' | \\
```

Comment:
This was messing up the cat command at the bottom of the script which was causing it to fail